### PR TITLE
Feature/44067

### PR DESCRIPTION
**Pull request by Aekal**
dev-44067
#384

### CODE REVIEW

File: components/02-elements/swatch/swatch.config.js [hunk 1]
```diff
@@ -27,51 +24,33 @@ module.exports = {
...
+            class: 'selected',
+            optionClass: 'swatch__option--image',
             attributes: 'style="background-image: url(../../images/swatch/swatch-red.jpg);"'
           },
           {
+            color: 'White',
+            optionClass: 'swatch__option--image',
             attributes: 'style="background-image: url(../../images/swatch/swatch-white.jpg);"'
           }
         ]
```
Comment (anqaka) [inline components/02-elements/swatch/swatch.config.js]: I've updated the config with aria attributes, please implement it where needed (views, modules)

File: components/03-modules/brief-info/brief-info.hbs [hunk 2]
```diff
@@ -2,19 +2,21 @@
...
-                {{{ item.title.attributes }}}
-                class="brief-info__title {{ item.title.class }}"
-            >
-               {{ item.title.text }}
-            </{{ item.title.tag }}>
-        </div>
+    <div class="brief-info__items-container">
+        {{#each items as |item|}}
+            <div class="brief-info__item">
+                <span class="brief-info__icon-wrapper {{ item.iconWrapperClass }}">
```
Comment (anqaka) [inline components/03-modules/brief-info/brief-info.hbs]: missing `aria-label` (there is no text, only icon inside

File: components/03-modules/brief-info/brief-info.hbs [hunk 3]
```diff
@@ -2,19 +2,21 @@
...
-            <{{ item.title.tag }}
-                {{{ item.title.attributes }}}
-                class="brief-info__title {{ item.title.class }}"
-            >
-               {{ item.title.text }}
-            </{{ item.title.tag }}>
-        </div>
+    <div class="brief-info__items-container">
+        {{#each items as |item|}}
+            <div class="brief-info__item">
```
Comment (anqaka) [inline components/03-modules/brief-info/brief-info.hbs]: maybe we can put it in list? b/c in fact we list here some features...

File: components/04-views/product-view/_product-view.scss [hunk 4]
```diff
@@ -260,13 +262,19 @@ $product-view__wrapper-margin\@medium                   : 0 0 40px 0 !default;
...
-    .swatch {
-        margin-bottom: $product-view__swatch-margin-bottom;
-        &__option {
-            min-width: $product-view__swatch-option-spacing;
-            min-height: $product-view__swatch-option-spacing;
+        .swatch {
+            margin-bottom: $product-view__swatch-margin-bottom;
+            &__option {
+                min-width: $product-view__swatch-option-spacing;
+                min-height: $product-view__swatch-option-spacing;
```
Comment (anqaka) [inline components/04-views/product-view/_product-view.scss]: interactives e,ements should be at least 48 x 48px of clickable area

File: components/02-elements/swatch/_swatch.scss [hunk 5]
```diff
@@ -4,6 +4,7 @@ $swatch__title-margin-bottom         : $spacer !default;
 $swatch__title-font-size             : $font-size-base !default;
 
 $swatch__option-size                 : 24px !default;
+$swatch__option-size--image          : 32px !default;
```
[the remark this answers is not in the document]
Comment (Aekal) [reply components/02-elements/swatch/_swatch.scss]: I made them bigger, each swatch has now 48x48px of clickable area.

File: components/03-modules/brief-info/brief-info.hbs (hunk 2 above)
Comment (Aekal) [reply components/03-modules/brief-info/brief-info.hbs]: Added

File: components/03-modules/brief-info/brief-info.hbs (hunk 3 above)
Comment (Aekal) [reply components/03-modules/brief-info/brief-info.hbs]: Done

File: components/04-views/product-view/_product-view.scss (hunk 4 above)
Comment (Aekal) [reply components/04-views/product-view/_product-view.scss]: Done

**Comment by Aekal** [comment]
Thanks for the readme, I've added accordingly aria attributes with script to simulate how do they work in magento.
I also added more variants of swatch: default, image, color (+ with labels) and I made some adjustments to `catalog-grid-item` (fixed spacing and added gray border - IMO looks better now).
Changelog updated.

**Comment by anqaka** [comment]
Great job! I don't have a pictures on local and I hear everything well 😍 👍

**Pull request merged**